server/code_switch/strategies/english_only_strategy.py

```diff
@@ -0,0 +1,10 @@
+from code_switch.strategies.cs_strategies import CSStrategy, CSParameters
+
+class EnglishOnlyStrategy(CSStrategy):
+	def __init__(self, cs_params: CSParameters=None):
+		CSStrategy.__init__(self)
+
+	@staticmethod
+	def predict_next_cs_level(current_cs_level=None):
```

Comment:
doesn't match super method signature

Reply:
Same as previous. I hope this solves the issue.